pkg/standalone/standalone.go

```diff
@@ -55,7 +55,7 @@ const (
 	ghcrURI                     = "ghcr.io/dapr"
 	daprGhcrImageName           = "dapr"
 	dockerContainerRegistryName = "dockerhub"
-	redisGhcrImageName          = "3rdparty/redis"
+	redisGhcrImageName          = "3rdparty/rejson"
```

Comment:
We cannot default to rejson because it is amd64 arch only. The redis image we have now support all architectures. We should default to redis and (optionally) allow override to rejson if needed.

Reply:
@artursouza The only thing that we can do here is let the user know that they have to use `rejson` for any Redis query based tasks and it will not be part of `dapr init`. Having a flag/env var override for a specific component goes against the portability that we want with dapr. Agreed that we by default init with Redis and Zipkin but having overrides as flags/env vars is not something we want to support. 

cc @yaron2 Any thoughts on this?

Reply:
Alos as a follow up, docker can still run amd64 images using emulation... but that in itself has issues wit hth rejson image ... where in the rejson container crashes on QEMU emulation on M1 Mac ...  https://github.com/redis/redis/issues/10162

Reply:
@artursouza we might have to revert this PR  -https://github.com/dapr/cli/pull/907/files as well then. Thoughts?

Reply:
LGTM